src/components/main/TodoList.tsx

```diff
@@ -3,6 +3,9 @@ import TodoListItem from "./TodoListItem";
 import { RemoveTodo, ToggleTodo, TodoItem } from "../../types/todo";
 import TEXTS from "../../constants/texts";
 import COLORS from "../../constants/color";
+import { memo, useCallback } from "react";
```

Comment:
```suggestion
import { useCallback } from "react";
```
`React.memo`가 전혀 사용되고 있지 않아서 이렇게 수정사항을 제안해 보았어요. 여기서 코멘트를 끝낼 수도 있겠지만 이러한 수정사항들을 쉽게 찾아낼 수 있는 환경 자체를 구축하는 것도 좋을 것 같아 보여요. 현재 미션에서는 필요하지 않았지만, 실제 배포하는 프로젝트라면 **빌드**를 대부분 할 텐데요, 이 빌드를 하기 위한 명령어의 앞에 ESLint를 통한 린트 작업을 먼저 한다든지, TypeScript를 이용해 먼저 컴파일해본다든지의 작업을 먼저 한다면 빌드 전 경고나 에러를 확인해 중요사항을 놓치지 않을 수 있을 것 같아요!

`package.json`의 스크립트들은 자유롭게 만드실 수 있으니 이러한 체크들을 차례대로 수행하는 스크립트도 한 번 만들어보셔도 좋을 것 같아요. `&&` 를 이용해 여러 스크립트들을 차례대로 수행할 수도 있어요. 아래는 예시에요.

```
"lint-and-compile": "npm run lint && npm run compile"
```

Reply:
기껏 타입스크립트와 eslint를 사용하면서 장점을 못 살리고 있었네요...
prettier가 eslint에 맞추어 오류를 띄울 줄 알았습니다.
이제 프로젝트 시작하기 전 `package.json` 도 신경을 써야할 것 같습니다!
수정했습니다!